src/js/controllers/index.js

```diff
@@ -341,6 +345,15 @@ angular.module('copayApp.controllers').controller('indexController', function($r
     $rootScope.$apply();
   });
 
+  $rootScope.$on('Local/NotFound', function(event) {
+    self.clientError = 'Check the Bitcore Wallet Service URL on Preferences';
```

Comment:
It could also be that URL was entered wrongly. The message should be: 
'Could not access to Bitcore Wallet Service: Service not found.'